concordances/test_fixtures.go

```diff
@@ -0,0 +1,107 @@
+package concordances
+
+//This file contains common fixtures shared by all test files in this package
```

Comment:
I do not know what is the solution in go. _test.go files cannot are not seen by other _test.files However, the this scaffolding is used in all tests - handlers, service & healthcheck. I will have to copy the same scaffolding in 3 files or make one gigantic test file for everything.